### PR TITLE
November Ratification Poll Outcomes

### PR DESCRIPTION
**Pull request by blimpa**
@Gala-G , please review 🥺

### CODE REVIEW

File: MIP86/MIP86.md [hunk 1]
```diff
@@ -9,11 +9,12 @@ Author(s): @CoinSharesCo
 Contributors: @jmmognetti
 Tags: PSM
 Type: General
-Status: Formal Submission
+Status: Rejected
 Date Proposed: 2022-09-13
-Date Ratified:
+Date Ratified: 2022-11-28
```
Comment (Gala-G) [inline MIP86/MIP86.md]: Erase date.

File: MIP87/MIP87.md [hunk 2]
```diff
@@ -8,12 +8,13 @@ Title: OUSD Decentralized Rewards
 Author(s): @pete
 Contributors: @joshfraser
 Tags: PSM
-Type:
-Status: Formal Submission
+Type: General
+Status: Rejected
 Date Proposed: 2022-10-04
-Date Ratified:
+Date Ratified: 2022-11-28
```
Comment (Gala-G) [inline MIP87/MIP87.md]: Erase date.

**Review by Gala-G — CHANGES_REQUESTED** [review]

File: MIP86/MIP86.md (hunk 1 above)
Comment (Flexgod69iner) [reply MIP86/MIP86.md]: Email

Comment (Flexgod69iner) [reply MIP86/MIP86.md]: Sorry I'm still wanting be involved I just need a mentor someone to guide me and show me how these things work I've looked at the guidelines and stuff for the maker forum there's a lack of communication please provide one on one training for those eager to learn but no trainees

File: MIP87/MIP87.md (hunk 2 above)
Comment (Flexgod69iner) [reply MIP87/MIP87.md]: How do I find out what mip I am

**Pull request merged**